README.md

```diff
@@ -62,6 +62,7 @@ import { Component } from '@angular/core';
 
 export class AppComponent {
   items: string[] = [{ name: "archie" }, { name: "jake" }, { name: "richard" }];
+  term = '';
```

Comment:
@arcsector I think it does not need! Can you reproduce it? It's optional I think.

Reply:
This should be necessary in the latest version of angular, the AOT compiler needs all properties defined in the Component.